src/types.ts

```diff
@@ -234,6 +235,17 @@ interface Applicant {
   listingId: number
 }
 
+interface Offer {
+  id: number
+  sentAt: Date
+  expiresAt: Date
+  answeredAt: Date
+  selectedApplicants: Array<Applicant>
+  status: OfferStatus
+  listingId: number
+  offeredApplicant: Applicant
+}
+
```

Comment:
```suggestion
interface Offer {
  id: number
  sentAt: Date | null
  expiresAt: Date
  answeredAt: Date | null
  selectedApplicants: Array<Applicant>
  status: OfferStatus
  listingId: number
  offeredApplicant: Applicant
}
```

Dessa två är nullable i databasen.
`answeredAt` för att det inte finns något svar än när erbjudandet skapas
`sentAt` för att erbjudandet inte nödvändigtvis har skickats än när det skapas (tänkte jag)

Reply:
Superbra! 🙏